### PR TITLE
feat: CHAIN-178 starkNetAddress Joi validator

### PR DESCRIPTION
**Pull request by Neozaru**
Related to https://darcs.atlassian.net/browse/CHAIN-178

### CODE REVIEW

**Review by adrian-gierakowski — DISMISSED** [review]

**Review by adrian-gierakowski — COMMENTED** [review]
Let’s bump minor version in package.json and the create a corresponding tag once merged to master

**Comment by Neozaru** [comment]
> Let’s bump minor version in package.json and the create a corresponding tag once merged to master

I don't have permission to merge / add tag I think

**Comment by adrian-gierakowski** [comment]
@Neozaru, is this still needed? If so then could you please bump the version to 1.2.0 so that it’s higher than on dev?

https://github.com/DeversiFi/dvf-utils/blob/89ba286d7b496e4d31c09461e66e905cb19d130e/package.json#L3

File: lib/isStarkNetAddress.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// TODO: Can probably be improve (ex: size of 64 is not mandatory but just usual)
+module.exports = value => /^0x[0-9abcdefABCDEF]{64}$/.test(value)
```
Comment (adrian-gierakowski) [inline lib/isStarkNetAddress.js:2]: If valid addresses can have size other than 64 then it seems to me that we should not force it

Comment (adrian-gierakowski) [reply lib/isStarkNetAddress.js:2]: @arijoon what do you think?

**Comment by adrian-gierakowski** [comment]
closing as not needed atm

**Pull request closed without merging**